task25/js/main.js

```diff
@@ -5,7 +5,89 @@ const modalPlace = document.getElementById("js-modalPlace");
 const closeModal = document.getElementById("js-modalClose");
 const closeModalButton = document.getElementById("js-modalCloseButton");
 const checkBox = document.getElementById("js-checkBox");
+const loginWrapper = document.getElementById("js-loginWrapper");
 const login = document.getElementById("js-login");
+const name = document.getElementById("name");
+const mail = document.getElementById("mail");
+const password = document.getElementById("password");
+
+const renderClassOfLoginWrapper = () => {
+    loginWrapper.classList.add("my-8", "mx-auto", "text-center", "w-52", "rounded-md", "bg-yellow-500", "p-3", "login-button")
+}
+renderClassOfLoginWrapper();
+
+const validationStatus = {
+    name : false ,
+    mail : false ,
+    password : false ,
+};
+
+const validations =  {
+    name : {
+        maxLength : 16,
+        minLength : 1,
+        validation : (value) => value.length > validations.name.minLength && value.length < validations.name.maxLength,
+        errorMessage : "ユーザー名は2文字以上15文字以下にしてください。",
+    } ,
+    mail : {
+        validation : (value) => /.+@.+\..+/.test(value),
+        errorMessage : "メールアドレスの形式になっていません。",
+    } ,
+    password : {
+        validation : (value) => /^(?=.*[A-Z])(?=.*[0-9])[a-zA-Z0-9.?\/-]{8,12}$/.test(value) ,
+        errorMessage : "8文字以上の大小の英数字を交ぜたものにしてください。",
+    }
+}
+
+const settingValidation = (isValid, target) => {
+    if(isValid) {
+        target.classList.add("isvalid");
+        target.classList.remove("invalid");
+        target.nextElementSibling.textContent = "";
+        validationStatus[target.id] = true;
+    } else {
+        target.classList.add("invalid");
+        target.classList.remove("isvalid");
+        target.nextElementSibling.textContent = validations[target.id].errorMessage;
+        validationStatus[target.id] = false;
+    }
+}
+
+const checkEventForValid = (e) => {
+    const target = e.target;
+    const value = target.value.trim();
+    const inputValue = validations[target.id].validation(value);
```

Comment:
IMO
I think it's a boolean that comes across here, not a value.
I think you should name the variable `is**.`